src/vma/sock/sockinfo_udp.cpp

```diff
@@ -469,6 +468,11 @@ int sockinfo_udp::bind(const struct sockaddr *__addr, socklen_t __addrlen)
 	on_sockname_change(name, *namelen);
 	si_udp_logdbg("bound to %s", m_bound.to_str());
 
+	if (m_rx_ring_map.size() == 1) {
+		rx_ring_map_t::iterator rx_ring_iter = m_rx_ring_map.begin();
+		m_p_rx_ring = rx_ring_iter->first;
```

Comment:
1. m_p_rx_ring is documented in sockinfo.h as designed for TCP only:
```
// we either listen on ALL system cqs or bound to the specific cq
ring*                   m_p_rx_ring; //used in TCP instead of m_rx_ring_map
```
Why does it appear in sockinfo_udp.cpp?

2. m_p_rx_ring is assigned here (and in line 2102) but it is not read read anywhere. Neither was it initialized. So it seems it is not used and can be removed